src/client/pages/ChangePassword.tsx

```diff
@@ -12,13 +12,47 @@ import { PageBody } from '@/client/components/PageBody';
 import { PageBodyText } from '@/client/components/PageBodyText';
 import { form, textInput, button } from '@/client/styles/Shared';
 import { SignInLayout } from '@/client/layouts/SignInLayout';
+import {
+  passwordValidation,
+  PasswordValidationComponent,
+} from '@/client/components/PasswordValidation';
+import { css } from '@emotion/core';
 
 export const ChangePasswordPage = () => {
   const { search } = useLocation();
   const globalState: GlobalState = useContext(GlobalStateContext);
   const { email = '', fieldErrors = [] } = globalState;
   const { token } = useParams<{ token: string }>();
 
+  const [password, setPassword] = useState('');
+  const [passwordRepeated, setPasswordRepeated] = useState('');
+
+  const serverErrorMessage: string | undefined = fieldErrors.find(
+    (fieldError) => fieldError.field === 'password',
+  )?.message;
+
+  const [validationErrorMessage, setValidationErrorMessage] = useState(
+    serverErrorMessage || '',
+  );
+  const hasSubmissionValidationError = validationErrorMessage !== '';
+
+  const passwordRepeatedErrorStyle = hasSubmissionValidationError
+    ? css`
+        border: 4px solid rgb(199, 0, 0);
+        color: rgb(199, 0, 0);
```

Comment:
with colours, we try to use the design system again as much as possible: http://theguardian.design/2a1e5182b/p/2668c8-code

`error.400` is what youre looking for: http://theguardian.design/2a1e5182b/p/492a30-light-palette